apps/coscrad-frontend/src/components/resources/audio-item/audio-item-detail.full-view.presenter.tsx

```diff
@@ -28,7 +33,58 @@ export const AudioItemDetailFullViewPresenter = ({
     text: plainText,
     name,
     actions,
+    annotations,
 }: ICategorizableDetailQueryResult<IAudioItemViewModel>): JSX.Element => {
+    const audioRef = useRef(null);
+
+    const [marks, setMarks] = useState<TimelineMark[]>([]);
+
+    useEffect(() => {
+        const updatedMarks = isNullOrUndefined(audioRef?.current?.currentTime)
+            ? []
+            : annotations.flatMap(({ connectedResources, name, id: noteId }) => {
+                  const timeRangeContext = connectedResources[0].context as ITimeRangeContext;
+
+                  const {
+                      timeRange: { inPointMilliseconds, outPointMilliseconds },
+                  } = timeRangeContext;
+
+                  const text = name.items.find(
+                      ({ role }) => role === MultilingualTextItemRole.original
+                  )?.text;
+
+                  const fullText = `[${inPointMilliseconds}] ${text} [${outPointMilliseconds}]`;
+
+                  /**
+                   * Should we break this logic out so we can share it for
+                   * video annotation?
+                   */
+                  return [
+                      buildTimelineMark({
```

Comment:
I would imagine this as just a vertical line that then becomes the left hand side of a rectangle when the time range is fully selected.  That way you can see the full selection and like we talked about it could have a transparency.